### PR TITLE
Add Brazilian-Portuguese translation

### PR DESCRIPTION
**Pull request by wederribas**
That's a first attempt to translate the app to PT-BR. Soon I can do the same for the About page as well.

P.S: Most of the changes were applied by prettier, hope that's ok

### CODE REVIEW

File: src/SubmitPage/SubmitForm.js [hunk 1]
```diff
@@ -64,8 +64,8 @@ const emptyData = {
   Locality: "",
   [PLACE_ID]: "",
   [STORE_ADDRESS]: "",
-  [OPENING_TIME]: null,
-  [CLOSING_TIME]: null,
+  [OPENING_TIME]: "",
+  [CLOSING_TIME]: "",
```
Comment (Siddharth11) [inline src/SubmitPage/SubmitForm.js:68]: This changes needs to be reverted. The time picker component doesn't not take a string. It either needs to be a date type object of null. We convert these values to string type while sending it back to backend

File: src/SubmitPage/SubmitForm.js [hunk 2]
```diff
@@ -249,6 +249,7 @@ class SubmitForm extends React.Component {
                 onChange={e => this.onChangeInput(e, STORE_ADDRESS)}
                 value={formData[STORE_ADDRESS]}
                 placeholder={translations.store_address_placeholder}
+                required
```
Comment (Siddharth11) [inline src/SubmitPage/SubmitForm.js:252]: Is this intended ?

File: src/SubmitPage/SubmitForm.js (hunk 1 above)
Comment (Siddharth11) [reply src/SubmitPage/SubmitForm.js:68]: Did you get an error with `null` as default value

File: src/SubmitPage/SubmitForm.js (hunk 2 above)
Comment (Siddharth11) [reply src/SubmitPage/SubmitForm.js:252]: @arjun27 is this a required field on backend too ?

Comment (arjun27) [reply src/SubmitPage/SubmitForm.js:252]: Address is not required anymore (after #202)

File: src/SubmitPage/SubmitForm.js (hunk 1 above)
Comment (wederribas) [reply src/SubmitPage/SubmitForm.js:68]: I was able to submit the form successfully for a location in Brazil. This change was made by prettier, so I guess we gotta exclude this rule from there. Anyway, that's easily revertible.

**Comment by Siddharth11** [comment]
@wederribas everything looks good! We can merge this PR once you rebase it with master. There is 1 merge conflict.

**Comment by arjun27** [comment]
@Siddharth11 - i've resolved the conflicts. feel free to merge if it looks good.

**Pull request merged**